runtime/queries/c-sharp/highlights.scm

```diff
@@ -1,47 +1,17 @@
 ;; Methods
-(method_declaration
```

Comment:
Most of the changes that we have in highlights.scm are improvements that haven't been upstreamed. Can you revert the changes to that file? It looks like the only changes necessary for updating the grammar come from the preprocessor change in https://github.com/tree-sitter/tree-sitter-c-sharp/pull/191:

```diff
diff --git a/runtime/queries/c-sharp/highlights.scm b/runtime/queries/c-sharp/highlights.scm
index 6b94dd8c..05eda6b9 100644
--- a/runtime/queries/c-sharp/highlights.scm
+++ b/runtime/queries/c-sharp/highlights.scm
@@ -228,8 +228,21 @@
   "let"
 ] @keyword
 
-(preprocessor_call) @keyword.directive
-(preprocessor_call (_) @keyword.directive)
+[
+  (if_directive (_) @keyword.directive)
+  (elif_directive (_) @keyword.directive)
+  (else_directive)
+  (endif_directive)
+  (warning_directive (_) @keyword.directive)
+  (error_directive (_) @keyword.directive)
+  (line_directive (_)? @keyword.directive)
+  (nullable_directive)
+  (define_directive (_) @keyword.directive)
+  (undef_directive (_) @keyword.directive)
+  (region_directive)
+  (endregion_directive)
+  (pragma_directive)
+] @keyword.directive
 
 ;; Linq
 (from_clause (identifier) @variable)
```

Reply:
Yeah, I just found out that by looking at the commits, I wrongly assumed that this file was in sync with the main repo, I'll try to add those directives as well.